docs/source/common_issues.rst

```diff
@@ -36,6 +36,13 @@ error:
 The second line is now fine, since the ignore comment causes the name
 ``frobnicate`` to get an implicit ``Any`` type.
 
+.. note::
+
+    The ``# type: ignore`` comment will only assign the implicit ``Any``
+    type if mypy cannot find information about that particular module. So,
+    if we did have a stub available for ``frobnicate`` then mypy will
```

Comment:
The tenses don't correspond -- either "do" instead of "did" or "would" instead of "will".